src/services/extensionV2Service.ts

```
@@ -0,0 +1,285 @@
+import {
+  Position,
+  Dimensions,
+  NodeVisual,
+  NodeType,
+  Execution,
+  WidgetValue,
+  WidgetIdentity,
+  WidgetContainer,
+  Connectivity,
+  SlotIdentity,
+  SlotConnection
```

ESLint

[error] 'SlotConnection' is defined but never used.

(unused-imports/no-unused-imports)